package.json

```
@@ -48,23 +48,23 @@
   "homepage": "https://github.com/cuaklabs/nestjs-meow#readme",
   "devDependencies": {
     "@types/jest": "^27.0.2",
-    "@typescript-eslint/eslint-plugin": "^5.4.0",
-    "@typescript-eslint/parser": "^5.4.0",
+    "@typescript-eslint/eslint-plugin": "^5.10.1",
+    "@typescript-eslint/parser": "^5.10.1",
     "enhanced-resolve": "^5.8.3",
-    "eslint": "^8.5.0",
+    "eslint": "^8.8.0",
     "eslint-plugin-import": "^2.25.3",
-    "eslint-plugin-jest": "^25.3.2",
+    "eslint-plugin-jest": "^26.0.0",
     "eslint-plugin-prettier": "^4.0.0",
     "husky": "^7.0.4",
-    "jest": "^27.4.5",
+    "jest": "^27.4.7",
     "lerna": "^4.0.0",
-    "lint-staged": "^12.1.3",
+    "lint-staged": "^12.3.2",
     "prettier": "^2.5.1",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
-    "ts-jest": "^27.1.2",
+    "ts-jest": "^27.1.3",
     "ts-node": "^10.4.0",
-    "typescript": "~4.5.4"
+    "typescript": "^4.5.5"
```

```suggestion
    "typescript": "~4.5.5"
```
Beware! Typescript often launches false minor updates